ehr/resources/web/ehr/panel/GeneticCalculationSettingsPanel.js

```diff
@@ -24,6 +24,10 @@ Ext4.define('EHR.panel.GeneticCalculationSettingsPanel', {
                 xtype: 'checkbox',
                 fieldLabel: 'Is Enabled?',
                 itemId: 'enabled'
+            },{
+                xtype: 'checkbox',
+                fieldLabel: 'Kinship validation?',
```

Comment:
Can we add a tooltip or other helper to give users a little more context about what this does?

Reply:
Added a tooltip